packages/state-transition/src/util/shuffle.ts

```diff
@@ -100,8 +102,7 @@ function innerShuffleList(input: Shuffleable, seed: Bytes32, dir: boolean): void
   }
 
   // Seed is always the first 32 bytes of the hash input, we never have to change this part of the buffer.
-  const _seed = seed;
-  Buffer.from(_seed).copy(buf, 0, 0, _SHUFFLE_H_SEED_SIZE);
+  buf.set(seed, 0);
```

Comment:
memory copy might be intentional here?

Reply:
It isn't intentional here. The update is fine, I think more eyes on this change will confirm that.

Reply:
confirm it's fine, maybe more clear to bound `seed` by `_SHUFFLE_H_SEED_SIZE` instead of 32 which will make it equivalent to the current implementation